api/types.proto

```diff
@@ -277,6 +277,9 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	// Preallocate tasks before shutting old ones down, not waiting for old to die
+	bool preAllocate = 3;
```

Comment:
Note that protocol buffers field names should be formatted like `pre_allocated`. They automatically get translated to CamelCase names in the autogenerated Go code.